WordPress/Classes/Utility/WPImmuTableRows.swift

```diff
@@ -273,15 +276,29 @@ struct ButtonRow: ImmuTableRow {
     static let cell = ImmuTableCell.class(WPTableViewCellDefault.self)
 
     let title: String
+    var textAlignment: NSTextAlignment = .center
+    var isLoading = false
     let action: ImmuTableAction?
+    var accessibilityIdentifier: String?
 
     func configureCell(_ cell: UITableViewCell) {
         cell.textLabel?.text = title
         cell.textLabel?.numberOfLines = 0
         cell.textLabel?.lineBreakMode = .byWordWrapping
 
         WPStyleGuide.configureTableViewActionCell(cell)
-        cell.textLabel?.textAlignment = .center
+        cell.textLabel?.textAlignment = textAlignment
+
+        if isLoading {
+            let indicator: UIActivityIndicatorView
+            indicator = UIActivityIndicatorView(style: .medium)
```

Comment:
Nitpick: merging the above two lines into one?

Reply:
Fixed. I copy-pasted it 🙃